src/Microsoft.Performance.Toolkit.Engine/Engine.cs

```diff
@@ -269,6 +278,30 @@ private set
             }
         }
 
+        /// <summary>
+        ///     Gets the collection of all enabled tables
+        /// </summary>
```

Comment:
for each of the new methods/properties, this should be added to the doc comments:

        /// <exception cref="ObjectDisposedException">
        ///     This instance is disposed.
        /// </exception>